x-pack/plugins/observability_solution/slo/public/embeddable/slo/error_budget/error_budget_react_embeddable_factory.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+import React, { useEffect } from 'react';
+import { Router } from '@kbn/shared-ux-router';
+import { createBrowserHistory } from 'history';
+import { ReactEmbeddableFactory } from '@kbn/embeddable-plugin/public';
+import {
+  initializeTitles,
+  useStateFromPublishingSubject,
+  fetch$,
+} from '@kbn/presentation-publishing';
+import { BehaviorSubject, Subject } from 'rxjs';
+import { KibanaContextProvider } from '@kbn/kibana-react-plugin/public';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import { SLO_ERROR_BUDGET_ID } from './constants';
+import { SloErrorBudgetEmbeddableState, SloEmbeddableDeps, ErrorBudgetApi } from './types';
+import { SloErrorBudget } from './error_budget_burn_down';
+
+export const getErrorBudgetPanelTitle = () =>
+  i18n.translate('xpack.slo.errorBudgetEmbeddable.title', {
+    defaultMessage: 'SLO Error Budget burn down',
+  });
+
+export const getErrorBudgetEmbeddableFactory = (deps: SloEmbeddableDeps) => {
+  const factory: ReactEmbeddableFactory<SloErrorBudgetEmbeddableState, ErrorBudgetApi> = {
+    type: SLO_ERROR_BUDGET_ID,
+    deserializeState: (state) => {
+      return state.rawState as SloErrorBudgetEmbeddableState;
+    },
+    buildEmbeddable: async (state, buildApi, uuid, parentApi) => {
+      const { titlesApi, titleComparators, serializeTitles } = initializeTitles(state);
+      const defaultTitle = getErrorBudgetPanelTitle();
+      titlesApi.setPanelTitle(state.title ?? defaultTitle);
```

Comment:
@ThomThomson Yep this is for the default title. I am not sure if I undertstand correctly how to do it. Let me take a look. Otherwise, I'll ping for more guidance.

Reply:
Here's what I did as part of this commit https://github.com/elastic/kibana/pull/181532/commits/813bbc50944fd34b92f6609009cc863c0273884a. Is this actually what you were looking for?

I tested it and works fine:
1) On initial Load it shows default title
2) On title change, it shows the saved title from the state
3) If I reset, I can the default title


https://github.com/elastic/kibana/assets/2852703/54ac2558-c120-42b7-bc6b-7a06aee5d87c

Reply:
I explored a bit more and here's my new solution using the defaultPanelTitle from PublishesPanelTitle interface https://github.com/elastic/kibana/pull/181532/commits/b9f0f4cdf121ddcf153c51ac518ee865905609a8. Here's the new behavior. Clicking on the reset button, shows the default title in Panel settings apart from panel header as previous video. Waiting on your feedback! 


https://github.com/elastic/kibana/assets/2852703/50b8b9f8-6281-4871-8c74-3016c9977084

Reply:
Yes this is the correct behaviour, and the implementation looks correct! One nit is that I don't think you need to serialize the `defaultPanelTitle`.